docs/content/credentials.md

```diff
@@ -32,8 +32,8 @@ like `{{ bundle.credentials.github_token }}` with the actual credential value
 before executing the step.
 
 Once the bundle finishes executing, the credentials are NOT recorded in the
-bundle instance (claim). Parameters are recorded there so that you can view them
-later using `porter instances show NAME --output json`.
+installation (claim). Parameters are recorded there so that you can view them
```

Comment:
This isn't strictly part of your change but it would be helpful to fix it here. 

It has to do with the larger features (rewriting claims), and the sentence will make more sense this way. Previously we stored a single file (called a claim) to represent an installation, but now we store many files to keep track of the history of what has happened to an installation. So we want to clarify that we won't store credentials in that history.

Would you please change this part?

```suggestion
installation history. Parameters are recorded there so that you can view them
```